python/chronos/src/setup.py

```diff
@@ -54,7 +54,7 @@ def setup_package():
         url='https://github.com/intel-analytics/BigDL',
         packages=get_bigdl_packages(),
         install_requires=['bigdl-orca=='+VERSION, 'bigdl-nano[pytorch]',
-                          'pandas==1.0.3', 'scikit-learn', 'statsmodels'],
+                          'pandas==1.0.5', 'scikit-learn', 'statsmodels'],
```

Comment:
maybe we just relax this version to be `pandas>=1.0.5`?

Reply:
done